runtime/actors/actorsystem.go

```diff
@@ -11,73 +11,43 @@ package actors
 
 import (
 	"context"
-	"fmt"
-	"os"
-	"strconv"
 	"time"
 
 	"github.com/hypermodeinc/modus/runtime/db"
 	"github.com/hypermodeinc/modus/runtime/logger"
+	"github.com/hypermodeinc/modus/runtime/messages"
 	"github.com/hypermodeinc/modus/runtime/pluginmanager"
 	"github.com/hypermodeinc/modus/runtime/plugins"
 	"github.com/hypermodeinc/modus/runtime/wasmhost"
 
 	goakt "github.com/tochemey/goakt/v3/actor"
-	goakt_static "github.com/tochemey/goakt/v3/discovery/static"
-	goakt_remote "github.com/tochemey/goakt/v3/remote"
-	"github.com/travisjeffery/go-dynaport"
 )
 
+const defaultAskTimeout = 10 * time.Second
```

Comment:
would calling it `default` confuse readers with the default set on the SDK methods (the one we just increased in PR #896)?

what do we think of calling it `internal`, since it's only used for internal ask() calls?
```suggestion
const internalAskTimeout = 10 * time.Second
```